benchmarks/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ function(ConfigureBench CMAKE_BENCH_NAME CMAKE_BENCH_SRC)
   set_target_properties(${CMAKE_BENCH_NAME} PROPERTIES POSITION_INDEPENDENT_CODE ON)
   target_include_directories(${CMAKE_BENCH_NAME} PRIVATE "$<BUILD_INTERFACE:${RMM_SOURCE_DIR}>")
   
-  target_link_libraries(${CMAKE_BENCH_NAME} benchmark pthread rmm)
+  target_link_libraries(${CMAKE_BENCH_NAME} benchmark::benchmark pthread rmm)
```

Comment:
Is the `benchmark::benchmark` target consistent whether the package is found or added?

Reply:
Yes, both spdlog and benchmark provide namespaced targets when used via `add_subdirectory`. gtest does not, that's why I added the aliases myself.

But I guess it brings up a good point, that is, the CI will not catch if things don't work with downloaded packages. You could add more CI checks that run with `CPM_DOWNLOAD_ALL=ON`, that would catch errors on that end.